InscryptionCommunityPatch/Card/AdditionalIcons.cs

```diff
@@ -0,0 +1,488 @@
+using HarmonyLib;
+using DiskCardGame;
+using UnityEngine;
+using System.Collections.Generic;
+
+namespace InscryptionCommunityPatch.Card;
+
+[HarmonyPatch]
+public class RenderAdditionalSigils
+{
+    // This patch modifies the way cards are rendered so that more than two sigils can be displayed on a single card
+    private static Transform Find(Transform start, string target = "CardBase")
+    {
+        foreach (Transform child in start)
+        {
+            if (child.name == target)
+            {
+                return child;
+            }
+
+            Transform result = Find(child, target);
+            if (result != null) 
+            {
+                return result;
+            }  
+        }
+        return null;
+    }
+
+    // private static void AddTripleIconSlotToCard(Transform abilityIconParent)
```

Comment:
Could this just be deleted instead of commented out? We can always look at git history if we need it again

Reply:
Resolved